linode/manager
# refactor: [M3-6956] - MUI v5 Migration - `SRC > Components > FormControlLabel`

## Description 📝
- Component refactor for `<FormControlLabel />`
 
## Major Changes 🔄
- Use named export ✏️
- Add Storybook story 📄
- Move out of `core` folder 🗂️

## Preview 📷
![Screenshot 2023-08-01 at 12 39 14 PM](https://github.com/linode/manager/assets/115251059/6d85da53-8cac-41e4-885a-0c9f21d3197d)

## How to test 🧪
- Verify automated checks pass ✅
- Check that form labels still look and work as expected
  - For example, go check the `Set Number of Nodes` section on the Database Create page
